.github/workflows/build.yml

```diff
@@ -115,7 +115,10 @@ jobs:
             "https://releases.hashicorp.com/packer/${PACKER_VERSION}/${PACKER_ZIP}"
           sudo unzip -d /opt/packer \
             ${{ env.CURL_CACHE_DIR }}/"${PACKER_ZIP}"
-          sudo mv /usr/local/bin/packer /usr/local/bin/packer-default
+          if [ -e /usr/local/bin/packer ]
```

Comment:
Would you please backport this to [cisagov/skeleton-generic](https://github.com/cisagov/skeleton-generic/blob/7ddea47402e48fbbcc470bdba96a627783f77e68/.github/workflows/build.yml#L117)? I believe this will start to be a problem as `ubuntu-latest` is starting to use `ubuntu-24.04` with a GA this month based on https://github.com/actions/runner-images/issues/9691#issuecomment-2063926929.